sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs

```diff
@@ -87,41 +87,28 @@ public async Task ListContainersSegmentAsync()
         public async Task ListContainersSegmentAsync_SecondaryStorageFirstRetrySuccessful()
         {
             var testExceptionPolicy = await this.PerformSecondaryStorageTest(1); // one GET failure means the GET request should end up using the SECONDARY host
-            this.AssertSecondaryStorageFirstRetrySuccessful(SecondaryStorageTenantPrimaryHost(), SecondaryStorageTenantSecondaryHost(), testExceptionPolicy);
+            this.AssertSecondaryStorageFirstRetrySuccessful(this.SecondaryStorageTenantPrimaryHost(), this.SecondaryStorageTenantSecondaryHost(), testExceptionPolicy);
```

Comment:
Why are you adding the `this.` in?  Doesn't the new .editorconfig tell you to remove them?